public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -147,9 +147,10 @@ export function useAllAlertRuleAbilities(rule: CombinedRule): Abilities<AlertRul
   const isProvisioned = isGrafanaRulerRule(rule.rulerRule) && Boolean(rule.rulerRule.grafana_alert.provenance);
   const isFederated = isFederatedRuleGroup(rule.group);
   const isGrafanaManagedAlertRule = isGrafanaRulerRule(rule.rulerRule);
+  const isPluginProvided = isPluginProvidedRule(rule);
```

Comment:
nit: I imagine that not only plugins will be providing an origin so maybe `isRuleWithOrigin` might make more sense? 

We can also move the logic to `isProvisioned` because we treat those pretty much the same – I don't feel strongly about this though so we can also just keep them separate for now.

Reply:
The way the code currently works not only reads checks the `__grafana_origin` presence but also it's value
If there is `__grafana_origin` label but not tied to any plugin then we don't treat such rules as managed externally.
So `isPluginProvidedRule` actually checks if its provided by a plugin, not if it has the `__grafana_origin` label
I changed the names of interfaces and functions to reflect that better